src/js/modules/breakpointHelper.js

```diff
@@ -41,7 +43,7 @@ export const getCurrentScreenWidth = () => {
 
     // Order the breakpoints from widest to narrowest,
     // takes the form [['narrow', '414px'], [...etc]]
-    const bps = [];
+    bps = [];
```

Comment:
Rather than pulling this out to be globally available, would be tidier to pull this out into a function that gets called and returns a set of breakpoints that can be called here and in `withinBreakpoint` when needed.